roles/pulibrary.nodejs/tasks/main.yml

```diff
@@ -1,28 +1,14 @@
 ---
-- name: Get upstream APT GPG key
-  apt_key:
-    id: "{{ nodejs__upstream_key_id }}"
-    keyserver: "hkp://pool.sks-keyservers.net"
-    state: present
-  register: pool_servers
-  ignore_errors: true
+- name: Download node apt update script
+  get_url:
+    url: https://deb.nodesource.com/setup_10.x
```

Comment:
👍 , but is this entirely secure?

Reply:
how do you mean here? @jrgriffiniii you want this checksummed?  other?

Reply:
Sorry, I meant that this is a cURL to a shell script hosted on the Web, it's not a blocking concern from me, please disregard.

Reply:
I am using the `apt-get` infrastructure now. I'd borrow from your other PR to use the `curl` -> `command`